frontend/public/locales/fr/renew-ita.json

```diff
@@ -20,9 +20,7 @@
       "confirm-email-required": "Confirmez l'adresse courriel",
       "email-required": "Entrez l'adresse courriel",
       "email-valid": "Entrez l'adresse courriel dans le bon format, par exemple nom@exemple.com",
-      "confirm-email-valid": "Entrez la confirmation de l'adresse courriel dans le bon format, par exemple nom@exemple.com",
-      "preferred-language-required": "Sélectionnez la langue de communication préférée",
-      "charactrs-valid": "Seuls les lettres, les chiffres et les caractères suivants sont acceptés\u00a0: apostrophe ('), virgule (,), point (.), trait d'union (-) et parenthèse ()"
+      "confirm-email-valid": "Entrez la confirmation de l'adresse courriel dans le bon format, par exemple nom@exemple.com"
```

Comment:
Bit of a nitpick, but I wonder if this should be `nom@example.com`. There's a reason we use `example.com` everywhere: https://en.wikipedia.org/wiki/Example.com

`exemple.com` is currently being squatted, but in the future it could be a malicious site.. we don't want to reference a potentially malicious site in the application.

Reply:
Fair point!